src/restic/node.go

```diff
@@ -14,10 +14,17 @@ import (
 
 	"runtime"
 
+	"bytes"
 	"restic/debug"
 	"restic/fs"
 )
 
+// Xattr is a tuple storing the xattr name and value.
+type Xattr struct {
```

Comment:
This name is not a good fit, what about `ExtendedAttribute`? Or even just `Attribute`?